lib/rules/linebreaks/index.js

```diff
@@ -81,23 +99,28 @@ function rule(actual, secondary, context) {
 			return data;
 		}
 
+		/**
+		 * @param {number} line
+		 * @param {number} column
+		 */
 		function reportNewlineError(line, column) {
 			// Creating a node manually helps us to point to empty lines.
 			const node = postcss.rule({
 				source: {
-					start: { line, column },
+					start: { line, column, offset: 0 },
+					input: new postcss.Input(''),
```

Comment:
[note] 

- [`Position`](https://postcss.org/api/#position) requires `offset`
- [`Source`](https://postcss.org/api/#source) requires `input`

...but, I'm not sure if the added dummy values are appropriate. 🤔

Reply:
I think it's acceptable. We can always revisit.